typescript/sdk/src/token/types.ts

```diff
@@ -147,6 +151,51 @@ export const WarpRouteDeployConfigSchema = z
     });
 
     return z.NEVER; // Causes schema validation to throw with above issue
+  })
+  // Verify that CCIP hooks are paired with CCIP ISMs
+  .transform((warpRouteDeployConfig, ctx) => {
+    // Get all the CCIP hooks and ISMs by origin->destination chain
+    const { ccipHookMap, ccipIsmMap } = getCCIPConfigMaps(
+      warpRouteDeployConfig,
+    );
+
+    // Check that each CCIP hook has a corresponding ISM
+    for (const [originChain, hooks] of Object.entries(ccipHookMap)) {
```

Comment:
IIuc, isn't `hooks` a map of destination chains to booleans? If so, calling it `hooks` feels off. Maybe `claimedDestinationsInHooks` or something like that?